docs/how_to_add_a_publisher.md

```diff
@@ -601,6 +605,15 @@ python -m scripts.generate_parser_test_files -p LATimes
 
 to generate a unit test for our parser.
 
+### Update tables
+
+To fully integrate your new publisher you have to add it to the [supported publishers](supported_publishers.md) table.
+You do so by simply running
+
+````shell
```

Comment:
```suggestion
```shell
```

Reply:
That doesn't matter as long as the count matches. Furthermore, we used four each in this hole section so I would leave it like that.